### PR TITLE
LPD-33278 'PrincipalException is null' when Reindexing a Object Entry with Translation

### PR DESCRIPTION
**Pull request by victorKammerer**
Jira ticket: [LPD-33278](https://liferay.atlassian.net/browse/LPD-33278)

[LPD-33278]: https://liferay.atlassian.net/browse/LPD-33278?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by victorKammerer** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/search/test/ObjectEntryIndexerReindexTest.java [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+			).put(
+				"textLocalized_i18n",
+				HashMapBuilder.put(
+					"en_US", "en_US " + RandomTestUtil.randomString()
+				).put(
+					"pt_BR", "pt_BR " + RandomTestUtil.randomString()
+				).build()
+			).build());
+
+		try (LogCapture logCapture = LoggerTestUtil.configureLog4JLogger(
```
Comment (linolaoj) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/search/test/ObjectEntryIndexerReindexTest.java:117]: Hi @victorKammerer , I see that in other test classes  the `try` of the `LogCapture` involves the whole method. Can we follow this here? Also, can you inline the methods that are used only once?

**Review by linolaoj — CHANGES_REQUESTED** [review]

**Comment by victorKammerer** [comment]
Done, @linolaoj!

**Review by linolaoj — APPROVED** [review]

**Comment by linolaoj** [comment]
ci:stop

**Comment by linolaoj** [comment]
ci:test:sf

**Comment by linolaoj** [comment]
ci:forward

**Pull request closed without merging**